docs/install/install-cave.md

```diff
@@ -68,17 +68,35 @@ Additionally users can choose to run a [virtual machine (VM)](#method-1-linux-vi
 
 **Latest Version: 18.2.1-6**
 
-For Windows, Unidata offers two installation options: a [**Linux Virtual Machine**](#method-1-linux-virtual-machine), or a [**Direct Windows Installation**](#method-2-direct-windows-install).
+For Windows, Unidata offers two installation options: a [**Direct Windows Installation**](#method-1-direct-windows-install), or a [**Linux Virtual Machine**](#method-2-linux-virtual-machine).
+
+!!! note "As of 4/24/2023, the direct install method has been completely simplified! No admin priviledges or extra software needed."
 
-Currently, the [virtual machine (VM)](#method-1-linux-virtual-machine) is the recommended form of install for those who do not have administrative priviledges on the machine, or beginners who want a simpler installation process.  
 !!! warning "At the moment, the VM option may not render all products in CAVE (ex. RGB composites of satellite imagery)"
 
-The [direct installation method](#method-2-direct-windows-install) is recommended for those who have administrative priviledges and a little bit of experience installing more software.
+### Method 1: Direct Windows Install
+
+This method has been simplified to include python packaged with CAVE so no other software installation is necessary. The CAVE shortcut on the Desktop is a `.bat` file that updates your env variables only when CAVE is launched, so it will not mess with any other python applications or environments.
```

Comment:
...python and java...

(I know we were bundling the JRE before, but we also said that was a requirement before, so just to avoid inclusion might be best to say java and python are now included)

Reply:
I'm not sure any of this is needed: "The CAVE shortcut on the Desktop is a `.bat` file that updates your env variables only when CAVE is launched, so it will not mess with any other python applications or environments."

Maybe just, "CAVE no longer needs any environment variable changes in order to run."

Reply:
Done